includes/admin/settings/register-settings.php

```diff
@@ -2173,7 +2173,10 @@ function edd_sendwp_callback($args) {
 
 	// Connection status partial label based on the state of the SendWP email sending setting (Tools -> SendWP)
 	$connected    = sprintf(
```

Comment:
Seems like we could just ditch the sprintf entirely? And just concatenate the different strings.